pkg/api/roles.go

```diff
@@ -40,6 +45,8 @@ var (
 	ScopeDatasourceID   = accesscontrol.Scope("datasources", "id", accesscontrol.Parameter(":id"))
 	ScopeDatasourceUID  = accesscontrol.Scope("datasources", "uid", accesscontrol.Parameter(":uid"))
 	ScopeDatasourceName = accesscontrol.Scope("datasources", "name", accesscontrol.Parameter(":name"))
+
+	ScopeTeamsID = accesscontrol.Scope("teams", "id", accesscontrol.Parameter(":teamId"))
```

Comment:
We just discussed that it might be a good idea to figure out if we can pull put these scopes and actions to it's own package (relevant discussion is also happening [here](https://github.com/grafana/grafana/pull/43951/files#r789525270)). This should not block the PR thought and can be done separately. 

As an alternative, if we don't want to go with a new package, we can put these in the models instead - this should allow us to reuse it elsewhere without facing any circular dependency problems.